spec/features/adding_peeps_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature "User adds a new peep" do
+
+  before(:each) do
+    User.create(:email => "test@test.com",
+                :password => 'test',
+                :password_confirmation => 'test')
+
+    scenario "when browsing the homepage" do
+      expect(Peep.count).to eq(0)
+      visit 'peeps/new'
+      add_peep("Bob")
+      expect(Peep.count).to eq(1)
+      peep = Peep.first
+      expect(peep.content).to eq("Bob")
+    end
```

Comment:
You're testing at the wrong level of abstraction here - feature tests should simulate user interaction with your app